### DISCUSSION TITLE
PowerShellPnP Won't Uninstall

### DISCUSSION THREAD

**Discussion by RCarlevaleSS** [pnp/powershell#717] [Q&A]
Removing the old version for it's sake of not working and trying to install PnP.Powershell instead of SharePointPnPPowerShellOnline. For whatever reason it says I don't have SharePointPnPPowerShellOnline though it clearly shows in the Get-Module list. 

Picture below.
[![Capture](https://user-images.githubusercontent.com/75378090/119523578-e01a9480-bd4a-11eb-866d-2577605ec37d.PNG)
](url)

**Comment by veronicageek** [comment]
I have transferred this as a question. This is not an issue with the PnP.PowerShell module per say.
You can delete the module manually in Uninstall Programs if it was installed with the .msi, and also under the correct path on your machine (under /WindowsPowerShell).

**Comment by LeonArmston** [comment]
I've had this before and as Veronique says uninstall the module from Uninstall Programs/Apps or delete the module folder. 

You can run the following PowerShell below to find where the module is installed.
```powershell
(Get-Module -Name "SharePointPnPPowerShellOnline").ModuleBase
```
Make a note of this folder and then close all instances of PowerShell down then delete this folder.
